demos/python_demos/image_inpainting_demo/image_inpainting_demo.py

```diff
@@ -33,30 +34,90 @@ def build_argparser():
                       help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is "
                            "acceptable. The demo will look for a suitable plugin for device specified. "
                            "Default value is CPU", default="CPU", type=str)
-    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask.", default=8, type=int)
-    args.add_argument("-mbw", "--max_brush_width", help="Optional. Max width of brush to draw mask.", default=24, type=int)
-    args.add_argument("-ml", "--max_length", help="Optional. Max strokes length to draw mask.", default=100, type=int)
-    args.add_argument("-mv", "--max_vertex", help="Optional. Max number of vertex to draw mask.", default=20, type=int)
-    args.add_argument("--no_show", help="Optional. Don't show output", action='store_true')
+    args.add_argument("-r", "--rnd", help="Optional. Use random mask for inpainting (with parameters set by -p, -mbw, -mk and -mv).Skipped in GUI mode", action='store_true')
+    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask. Skipped in GUI mode", default=8, type=int)
+    args.add_argument("-mbw", "--max_brush_width", help="Optional. Max width of brush to draw mask. Skipped in GUI mode", default=24, type=int)
+    args.add_argument("-ml", "--max_length", help="Optional. Max strokes length to draw mask. Skipped in GUI mode", default=100, type=int)
+    args.add_argument("-mv", "--max_vertex", help="Optional. Max number of vertex to draw mask. Skipped in GUI mode", default=20, type=int)
+    args.add_argument("-mc", "--mask_color", help="Optional. Color to be treated as mask (provide 3 RGB components in range of 0...255). Default is 0 0 0. Skipped in GUI mode", default=0, type=int, nargs="+")
```

Comment:
Line too long. Recommended line length is 120. 
In this case good option is to use individual lines for description, e.g:
```python
    args.add_argument("-mv", "--max_vertex", default=20, type=int,
                      help="Optional. Max number of vertex to draw mask. "
                           "Skipped in GUI mode")
```